lib/rubocop/cop/style/accessor_grouping.rb

```diff
@@ -41,6 +41,8 @@ class AccessorGrouping < Cop
         ACCESSOR_METHODS = %i[attr_reader attr_writer attr_accessor attr].freeze
 
         def on_class(node)
+          return if grouped_style? && !accessors_without_comments(node)
```

Comment:
This check seems not needed.

Reply:
Right. I was thinking it might be needed when there are multiple separate accessors and only one without comment. But that case also seems to be handled by `check`. Removed this.